virtcontainers/capabilities.go

```diff
@@ -9,6 +9,8 @@ const (
 	blockDeviceSupport = 1 << iota
 	blockDeviceHotplugSupport
 	multiQueueSupport
+	hotplugUnsupported
+	plan9FSUnsupported
```

Comment:
@egernst instead of calling this is9pSupported, should this not be something more generic. 9p is one way to share files between host and the VM?